org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/corext/template/java/JavaContextTypeRegistry.java

```diff
@@ -25,9 +25,17 @@
 public class JavaContextTypeRegistry extends ContextTypeRegistry {
 
 	public JavaContextTypeRegistry() {
-		JavaContextType contextType = new JavaContextType();
-		contextType.initializeContextTypeResolvers();
-		addContextType(contextType);
+		JavaContextType contextTypeStatements = new JavaContextType(JavaContextType.ID_STATEMENTS);
```

Comment:
Yes, otherwise `contextType = (JavaContextType) JavaLanguageServerPlugin.getInstance().getTemplateContextRegistry().getContextType(JavaContextType.ID_MEMBERS)` would be null (for ID_ALL as well).